### PR TITLE
solution

### PR DESCRIPTION
**Pull request by AVyha**

### CODE REVIEW

**Review by Andriy-Makarenko — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int):
```
[the remark this answers is not in the document]
Comment (Andriy-Makarenko) [reply app/main.py:4]: Ok. Good job

**Review by Andriy-Makarenko — APPROVED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+        self.husband = husband
+
+
+def create_person_list(people_list: list) -> list:
+    people = []
+    people.extend(people_list)
+    for index in range(len(people)):
+        people[index] = Person(people[index]['name'], people[index]['age'])
+
+    for person in range(len(people_list)):
```
Comment (Y-Havryliv) [inline app/main.py:22]: better to rename `person` to `index`

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,34 @@
...
+
+    def husband_add(self, husband) -> None:
+        self.husband = husband
+
+
+def create_person_list(people_list: list) -> list:
+    people = []
+    people.extend(people_list)
+    for index in range(len(people)):
+        people[index] = Person(people[index]['name'], people[index]['age'])
```
Comment (Y-Havryliv) [inline app/main.py:20]: please, use double quotes everywhere

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
-    pass
+    def wife_add(self, wife) -> None:
+        self.wife = wife
+
+    def husband_add(self, husband) -> None:
+        self.husband = husband
+
+
+def create_person_list(people_list: list) -> list:
+    people = []
```
Comment (Y-Havryliv) [inline app/main.py:17]: btw you can do it in a more simple way using list comprehension

**Review by Y-Havryliv — CHANGES_REQUESTED** [review]

**Review by Y-Havryliv — APPROVED** [review]

**Pull request closed without merging**